src/sentry/tasks/codeowners/__init__.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+from django.conf import settings
+
+logger = logging.getLogger("sentry.tasks.code_owners")
+
+__all__ = (
+    "code_owners_auto_sync",
+    "update_code_owners_schema",
+    "logger",
+)
+
+_tasks_list = ("code_owners_auto_sync", "update_code_owners_schema")
+settings.CELERY_IMPORTS += tuple(f"sentry.tasks.code_owners.{task}" for task in _tasks_list)
```

Comment:
@NisanthanNanthakumar IMO it would be cleaner to just add the tasks in src/sentry/conf/server.py. If we had a bunch of tasks doing it programmatically like this might be better but with only two entries seems like overkill. It takes more characters to do it this way lol.